src/EditorFeatures/Test/MetadataAsSource/MetadataAsSourceTests.VisualBasic.cs

```diff
@@ -126,101 +126,47 @@ End Class
 
 using System.Runtime.InteropServices;
 
-namespace System
+namespace System;
+
+[Serializable]
+[AttributeUsage(AttributeTargets.Class | AttributeTargets.Struct | AttributeTargets.Enum | AttributeTargets.Constructor | AttributeTargets.Method | AttributeTargets.Property | AttributeTargets.Field | AttributeTargets.Event | AttributeTargets.Interface | AttributeTargets.Delegate, Inherited = false)]
+[ComVisible(true)]
+public sealed class [|ObsoleteAttribute|] : Attribute
 {{
-    //
-    // Summary:
-    //     Marks the program elements that are no longer in use. This class cannot be inherited.
-    [Serializable]
-    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Struct | AttributeTargets.Enum | AttributeTargets.Constructor | AttributeTargets.Method | AttributeTargets.Property | AttributeTargets.Field | AttributeTargets.Event | AttributeTargets.Interface | AttributeTargets.Delegate, Inherited = false)]
-    [ComVisible(true)]
-    [__DynamicallyInvokable]
-    public sealed class [|ObsoleteAttribute|] : Attribute
-    {{
-        private string _message;
-
-        private bool _error;
-
-        //
-        // Summary:
-        //     Gets the workaround message, including a description of the alternative program
-        //     elements.
-        //
-        // Returns:
-        //     The workaround text string.
-        [__DynamicallyInvokable]
-        public string Message
+    public string Message
+    {{
+        get
         {{
-            [__DynamicallyInvokable]
-            get
-            {{
-                return _message;
-            }}
+            /*Error: Empty body found. Decompiled assembly might be a reference assembly.*/
```

Comment:
This part of the output isn't really the point of the test, but any idea why this might have changed?

Reply:
yeah, def odd.